documentation/dotnet-trace-instructions.md

```diff
@@ -0,0 +1,131 @@
+# Trace for performance analysis utility (dotnet-trace)
+
+The dotnet-trace tool is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It is built around the EventPipe technology of the .NET Core runtime as an alternative to ETW on Windows and LTTNG on non-Windows.
+
```

Comment:
It would be good to describe what users can do with the trace once they have it, ie load it in speedscope or Perfview. In the future we'd add Visual Studio.